### ISSUE TITLE
zar import failure for a 14GB file

### ISSUE THREAD

**Issue by marktwallace** [brimsec/zq#1731]
This bug is a pain to reproduce, since I only got it more than halfway into importing a 14GB file that usually takes over an hour.

AFAIK, I did not cancel this myself (no ctrl-C). I did suspend the task more than once, with ctrl-Z and resumed with "fg %1".

Phil thought Matt should look at this first, so I assigned it.

$ zar import -R s3://zqd-demo-1/mark/zeek-logs/conn s3://brim-sampledata/wrccdc/zeek-logs/conn.log.gz
MultipartUpload: upload multipart failed
	upload id: q.BAZsXLc9yFb6sb21i7qyN12gJuiSJqxaPZIkqa_2H4cEkkX89Rmew_K0.kh_Q4hNLKQsq7HD1bDft72kMMWNFQ7otenBam6txNVnW3iJSQUH2Ow61jgzpx3HpZaddX
caused by: RequestCanceled: request context canceled
caused by: context canceled

BTW, I am trying this over again now.

**Comment by marktwallace** [comment]
This did not happen again when I retried it, so I think it is safe to assume it is pilot error and something in my environment cancelled the zar job.

**Comment by philrz** [comment]
@marktwallace: Did you also try repeating the ctrl-Z and resumes in your retry? If we can only repro the issue under those conditions, it might not be something we could do much about, but might also be worth disclosing to users. I'm somewhat reminded of a problem we noticed in the Brim app where an import breaks if a user closes their laptop lid and suspends the whole system and then opens it back up again. We found this fails reliably, but in the past we it showed a generic/confusing error message when it happened. So once we identified the root cause we made the error message more targeted, along the lines of "Don't close your laptop lid during imports ". 😄
